src/pg_to_evalscript/javascript_processes/sort.js

```diff
@@ -0,0 +1,50 @@
+function sort(arguments) {
+  const { data, asc = true, nodata = null } = arguments;
+  let newData = [...data];
+  let arr_of_nulls = newData.filter((x) => x === null);
+
+  newData = newData.filter((x) => x !== null);
```

Comment:
This can be refactored to

```javascript
let newData = [...data].filter((x) => x !== null);
let arr_of_nulls = [...data].filter((x) => x === null);
```

I also think a check if `data` is array and if `asc` and `nodata` are either boolean or `null` should be added at the beginning.

Reply:
Maybe there should also be a check for every element of the array (can be number, string or `null`, but not boolean, array, etc.)

Reply:
Optimization to go through the array less times (once instead of 2 or 3 times)

- go through the array with the `for` loop
- check if the element is of appropriate type
- check if it's `null`
  - if it is, add it to `arr_of_nulls`
  - else, add it to the `newData`

Reply:
Should be fixed.

Reply:
Thanks, looks good.